docs/apache-airflow/security/webserver.rst

```diff
@@ -313,9 +313,9 @@ setting the ``RATELIMIT_*`` configuration settings in ``webserver_config.py``.
 For example, to use Redis as a rate limit storage you can use the following configuration (you need
 to set ``redis_host`` to your Redis instance)
 
-```
-RATELIMIT_STORAGE_URI = 'redis://redis_host:6379/0
-```
+.. code-block:: ini
+
+    RATELIMIT_STORAGE_URI = 'redis://redis_host:6379/0
```

Comment:
Probably not related to this PR, but is the dangling single quote correct?

Reply:
yep, this should be python instead

Reply:
just fixed it. Thanks!